src/app/contexts/ToggleContext/index.jsx

```diff
@@ -1,16 +1,63 @@
-import React, { createContext, useReducer } from 'react';
-import { node } from 'prop-types';
-import toggleReducer from '../../reducers/ToggleReducer';
+import React, { createContext, useEffect, useReducer } from 'react';
+import { node, string } from 'prop-types';
+import webLogger from '#lib/logger.web';
+import { toggleReducer, updateToggles } from './reducer';
 import defaultToggles from '#lib/config/toggles';
+import constructTogglesEndpoint from './utils/constructTogglesEndpoint';
 
+const logger = webLogger();
 const ToggleContext = createContext({});
 
-const ToggleContextProvider = ({ children }) => {
+const ToggleContextProvider = ({ children, service, origin }) => {
+  const simorghToggles = defaultToggles[process.env.SIMORGH_APP_ENV || 'local'];
   const [toggleState, toggleDispatch] = useReducer(
     toggleReducer,
-    defaultToggles,
+    simorghToggles,
   );
 
+  const environment = process.env.SIMORGH_APP_ENV || 'local';
```

Comment:
We could move this `environment` variable above line 12 and used there.

Reply:
Thanks. Done.